modules/nw-egress-ips-about.adoc

```diff
@@ -149,6 +149,24 @@ On Azure, the following capacity limits exist for IP address assignment:
 
 For more information, see link:https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits?toc=/azure/virtual-network/toc.json#networking-limits[Networking limits].
 
+[id="nw-egress-ips-multi-nic-considerations_{context}"]
+== Considerations for assigning non-OVN managed networks to egress IP addresses
+
+Egress IP addresses can be set within _non-OVN managed networks_, or networks that are not managed by Open Virtual Network (OVN). These networks provide users with the option to route workload traffic over specific network interface cards (NICs) for purposes such as traffic segmentation or meeting specialized requirements.
+
+For users who want to assign non-OVN managed networks to their egress IP addresses, the following conditions must be met:
+
+* {product-title} is installed on a bare metal cluster. Assigning non-OVN managed networks to egress IP addresses is disabled in cloud and hypervisor environments, and does not work for host-networked pods. 
+* The network link should be operational. 
+* IP addresses must have a universal, or _global_, scope. 
+* Links and their addresses must not be removed during runtime after an egress IP address is assigned to it. If you want to remove the link, you must first remove the egress IP address and then remove the address or network link. 
+* IP forwarding must be enabled for the network link. To enable IP forwarding, you can set `ipForwarding: Global` in your `config.yaml` file. 
```

Comment:
Please link to the relevant config.yaml

Reply:
@djuran Went ahead and added the YAML snippet.

Reply:
You still don't specify which config.yaml file you are referring to